proto/xudrpc.proto

```diff
@@ -732,6 +743,15 @@ message RemoveOrderResponse {
   uint64 quantity_on_hold = 1 [json_name = "hold"];
 }
 
+message CancelAllOrdersRequest {}
+
+message CancelAllOrdersResponse {
+  // The order ids that are successfully removed.
+  repeated string removed_orders = 1 [json_name = "removed_orders"];
+  // The order ids that are failed to be removed.
+  repeated string failed_orders = 2 [json_name = "failed_orders"];
```

Comment:
I don't think this field is necessary. There shouldn't be a reason that we are unable to remove/cancel our own orders, they'll be canceled and if not there is something seriously wrong.

Reply:
Alright removing then